tendermint/src/validator.rs

```diff
@@ -277,7 +279,7 @@ tendermint_pb_modules! {
             RawValidatorSet {
                 validators: value.validators.into_iter().map(Into::into).collect(),
                 proposer: value.proposer.map(Into::into),
-                total_voting_power: value.total_voting_power.into(),
+                total_voting_power: value.total_voting_power.unwrap().into(),
```

Comment:
This should perhaps convert a `None` into the default value for the field (0).

Reply:
Done in eab906d1b1287b42d24e552a10876e9357806b36